server/src/main/java/com/cloud/network/security/SecurityGroupManagerImpl.java

```diff
@@ -858,6 +859,10 @@ public SecurityGroupVO createSecurityGroup(CreateSecurityGroupCmd cmd) throws Pe
         Account caller = CallContext.current().getCallingAccount();
         Account owner = _accountMgr.finalizeOwner(caller, cmd.getAccountName(), cmd.getDomainId(), cmd.getProjectId());
 
+        if (name.trim().equals("")) {
```

Comment:
Maybe use the commons check ifNotEmpty? We use this on multiple locations in the CloudStack code

Reply:
Updated the code